matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/verification/VerificationTransportToDevice.kt

```diff
@@ -76,12 +76,12 @@ internal class VerificationTransportToDevice(
                 .configureWith(SendToDeviceTask.Params(MessageType.MSGTYPE_VERIFICATION_REQUEST, contentMap)) {
                     this.callback = object : MatrixCallback<Unit> {
                         override fun onSuccess(data: Unit) {
-                            Timber.v("## verification [$tx.transactionId] send toDevice request success")
+                            Timber.v("## verification [${tx?.transactionId}] send toDevice request success")
```

Comment:
> I don't really understand the changes, why are they needed?

@ganfra in the new version, only the `transactionId` content will be logged and not `tx` content (i.e. the class name followed by its memory address, but in the case of a data class, the whole class instance content) followed by the string ".transactionId". The new check will prevent more annoying errors, fixed for instance by #6706.

Reply:
Ok thanks for the explanation!